tutorials/run-spring-petclinic-on-app-engine-cloudsql.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Run Spring Pet Clinic with Cloud SQL on Google App Engine flexible Environment
+description: Learn how to deploy Spring Boot Pet Clinic application to Google App Engine flexible environment and use Cloud SQL.
+author: jabubake
+tags: App Engine, Cloud SQL, Spring Boot
+date_published: 2017-05-12
+---
+
+This tutorial will walk you through getting the Spring Pet Clinic application up and running on App Engine flexible environment with Cloud SQL.
+
+### Spring Boot and Spring Pet Clinic
+[Spring Boot](https://projects.spring.io/spring-boot/) provides Java developers a quick, annotation driven way to deploy services using minimal code.
+[Spring Pet Clinic](https://github.com/spring-projects/spring-petclinic) is a popular Spring application that will quickly demonstrate the power of Spring Boot.
+
+### App Engine flexible environment for Java
+[App Engine flexible environment](https://cloud.google.com/appengine/docs/flexible/java/) provides the ability to run your Java applications in Docker containers on Google Compute Engine machines, with the ability
+to have them deployed, monitored and auto-scaled.
+We will be using the [Java 8 runtime environment](https://cloud.google.com/appengine/docs/flexible/java/dev-java-only) for this tutorial.
+
+### Cloud SQL
+[Cloud SQL for MySQL](https://cloud.google.com/sql/docs/mysql/) is a fully managed MySQL database service on Google Cloud Platform (GCP).
+
+### Prerequisites
+1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com/).
+1. Enable billing for your project.
+1. Install the [Google Cloud SDK](https://cloud.google.com/sdk/).
+
+### Prepare
+1. Initialize the Cloud SDK, create an App Engine application, and authorize the Cloud SDK for using GCP APIs in your local environment:
+```
+   gcloud init
+   gcloud app create
+   gcloud auth application-default login
+```
+1. Clone and test the Pet Clinic application locally:
```

Comment:
This second 1. isn't rendering as a 2. I think the code blocks need to be indented 4 spaces.